rm1/src/components/Counter.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+
+//react class component
+export default class Counter extends Component {
+    constructor(){
+        super()
+    }
+
+state = {
+    numberCounter: 0
+}
+
+ IncreaseCount = () => {
```

Comment:
Consider avoiding unnecessary upper case since in JS capital letter is reserved for classes or constructor functions.